packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -291,6 +293,9 @@ class CupertinoPageRoute<T> extends PageRoute<T> with CupertinoRouteTransitionMi
     assert(opaque);
   }
 
+  @override
+  DelegatedTransitionBuilder? get delegatedTransition => CupertinoPageTransition.delegateTransition;
```

Comment:
I've mentioned this before but we should decide whether it's "delegate" or "delegated" and make sure to use the same name everywhere.